app/controllers/user_topics_controller.rb

```diff
@@ -0,0 +1,63 @@
+require 'erb'
+require 'sinatra/flash'
+require 'pony'
+require 'json'
+
+module Sinatra
+  module App
+    module UserTopicController
+
+      def self.registered(app)
+
+        app.get '/dashboard' do
+          check_authentication
+          user_id = current_user.id
+          @user= User.find(user_id)
```

Comment:
Do we need to go to the database and retrieve the user? I thought your `current_user` method returns a user, right?